bin/leanc.in

```diff
@@ -1,48 +1,30 @@
 #!/usr/bin/env bash
 # Lean compiler
 #
-# A simple wrapper around C and C++ compilers. Defaults to the compiler Lean was built with,
-# which can be overridden with the environment variables `LEAN_CC` and `LEAN_CXX`. All parameters are passed
-# as-is to the wrapped compiler.
+# A simple wrapper around a C++ compiler. Defaults to the compiler Lean was built with,
+# which can be overridden with the environment variable `LEAN_CXX`. All parameters are passed
+# as-is to the wrapped compiler. Note that we use a C++ compiler purely for linking against
+# the C++ standard library Lean depends on, all input files are treated as C code.
```

Comment:
@leodemoura minor change of plans, it turns out that using `clang++` is easier after all